pkgs/applications/networking/ssb/patchwork/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, appimage-run, fetchurl, runtimeShell, gsettings-desktop-schemas, gtk3, gobject-introspection, wrapGAppsHook }:
+
+let
+  # latest version that runs without errors
+  # https://github.com/ssbc/patchwork/issues/972
+  version = "3.11.4";
+in
+
+stdenv.mkDerivation rec {
+  name = "patchwork-${version}";
```

Comment:
I think the new best practice is to just use `pname = "patchwork"`

Reply:
```
error: derivation name missing
```

Reply:
You also need `inherit version;` below.